cms/models.py

```diff
@@ -27,20 +27,18 @@
     FORM_FIELD_CHOICES,
     AbstractForm,
     AbstractFormField,
-    AbstractFormSubmission,
 )
 from wagtail.contrib.routable_page.models import RoutablePageMixin, route
 from wagtail.coreutils import WAGTAIL_APPEND_SLASH
 from wagtail.embeds.embeds import get_embed
 from wagtail.embeds.exceptions import EmbedException
 from wagtail.fields import RichTextField, StreamField
 from wagtail.images.models import Image
-from wagtail.models import Orderable, Page, Site
+from wagtail.models import Page
 from wagtail.search import index
 
 from cms.blocks import (
     CourseRunCertificateOverrides,
-    FacultyBlock,
```

Comment:
Looks like FacultyBlock is no longer used anywhere, probably can be removed as well. But up to you

Reply:
Nice catch.  I created an issue for it here: https://github.com/mitodl/mitxonline/issues/1855